Master-OB-OpenAPI.json

```diff
@@ -2442,7 +2441,7 @@
             "x-ob-item-type": "stdi:dateItemType",
             "x-ob-usage-tips": "",
             "x-ob-sample-value": {
-              "Value": "1969"
+              "Value": "1969-01-01"
```

Comment:
A year-only date is (should be) a valid value. I don't think we want to change the element to "DateBuilt".